lib/xclarity_client/services/update_policy_management.rb

```diff
@@ -0,0 +1,74 @@
+require 'json'
+
+module XClarityClient
+  class UpdatePolicyManagement < XClarityBase
+
+    include XClarityClient::ManagementMixin
+
+    def initialize(conf)
+      super(conf, UpdatePolicy::BASE_URI)
+    end
+
+    def population(opts = {})
```

Comment:
Please, use the `fetch_all` method defined on `Services::XClarityService` instead `population`

Reply:
thanks.. i have made respective update